{"uuid":"6703caf7-230d-42c0-a0c7-7932cf9d7672","historyId":"394c98f1be779e569b761280d5f6b807","fullName":"Tests.ChangePassPageTest.TC_PSA_0075","labels":[{"name":"package","value":"Tests.ChangePassPageTest"},{"name":"testClass","value":"Tests.ChangePassPageTest"},{"name":"testMethod","value":"TC_PSA_0075"},{"name":"parentSuite","value":"Default Suite"},{"name":"suite","value":"SAPortalTest_Selenium_POM"},{"name":"subSuite","value":"Tests.ChangePassPageTest"},{"name":"host","value":"DESKTOP-MCJ49E3"},{"name":"thread","value":"8316@DESKTOP-MCJ49E3.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"}],"links":[],"name":"Validation of expired old password after change password","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"The following asserts failed:\n\tError:Despite clicking on change pass button the Change Password screen is still open expected [false] but found [true],\n\tThe API Error message \" Something wrong!! Please try again\" appears expected [false] but found [true]","trace":"java.lang.AssertionError: The following asserts failed:\n\tError:Despite clicking on change pass button the Change Password screen is still open expected [false] but found [true],\n\tThe API Error message \" Something wrong!! Please try again\" appears expected [false] but found [true]\r\n\tat org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:43)\r\n\tat Tests.ChangePassPageTest.TC_PSA_0075(ChangePassPageTest.java:918)\r\n\tat java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:577)\r\n\tat org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)\r\n\tat org.testng.internal.Invoker.invokeMethod(Invoker.java:583)\r\n\tat org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)\r\n\tat org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)\r\n\tat org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)\r\n\tat org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:648)\r\n\tat org.testng.TestRunner.run(TestRunner.java:505)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:455)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:364)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1137)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1049)\r\n\tat org.testng.TestNG.run(TestNG.java:1017)\r\n\tat com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)\r\n\tat com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:109)\r\n"},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1667386362487,"stop":1667386365540}